builders/manage/developer-portal/smart-contracts-creation-filter.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Smart EVM - Whitelist Smart Contracts Deployment
```

Comment:
```suggestion
title: Smart EVM - Whitelisted Smart Contracts Deployment
```

Reply:
"Whitelist" is intended as a verb here (the other titles in the section start with verbs like manage, dispatch, enable, etc)

Reply:
Then I would say "Whitelist Smart Contract Deployments"